File: exercises/small_problems_101/easy_1/reverse_it_2.rb
def reverse_words(string)
  output = []
  string.split(' ').each do |word|
    if word.length >=5
      output.push(word.reverse!)
    else
      output.push(word)
    end
  end
  output.join(' ')
end

puts reverse_words('Professional')          # => lanoisseforP
puts reverse_words('Walk around the block') # => Walk dnuora the kcolb
puts reverse_words('Launch School')         # => hcnuaL loohcS

File: frontend/javascript/exercises/easy_5/reverse_it_2.js
function reverseWords(words) {
  var wordsArray = words.split(' ');
  var reverseWordArray = wordsArray.map(function (word) {
    if (word.length >= 5) {
      return word.split('').reverse().join('')
    } else {
      return word;
    }
  });
  return reverseWordArray.join(' ');
}

console.log(reverseWords('Professional') === 'lanoisseforP');          // lanoisseforP
console.log(reverseWords('Walk around the block') === 'Walk dnuora the kcolb'); // Walk dnuora the kcolb
console.log(reverseWords('Launch School') === 'hcnuaL loohcS');         // hcnuaL loohcS